Makefile

```diff
@@ -15,7 +15,7 @@ govet: goimports
 
 golint: govet
 	go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.39.0
-	golangci-lint run
+	golangci-lint run --enable-all --exclude-use-default=false -D errcheck -D gomnd
```

Comment:
looks like `--enable-all` [is deprecated](https://golangci-lint.run/usage/install/#other-ci).

Reply:
Weird, when I removed the enable-all flag and ran with verbose output it looked like no linters were being run, but I guess we're just passing the default linters (but not all of them, when you run them all using --enable-all)